Main/src/Coffee/Action.java

```diff
@@ -0,0 +1,165 @@
+package Coffee;
+
+import java.util.Scanner;
+
+public class Action {
+    public static int water=400;
+    public static int milk=540;
+    public static int coffeeBeans=120;
+    public static int dcups=9;
+    public static int money=550;
+
+    public static void getAction() {
+        System.out.println("Step 1");
+        Scanner in = new Scanner(System.in);
+        System.out.println("Please enter required action: 'buy', 'fill' or 'pull'");
+        String actionType = in.nextLine();
+        if (actionType.equals("fill")) {
```

Comment:
@GayaneGulanyan jan you can use swithch case for practice

Reply:
done